library/src/morpheus-master/sass/modules/tool/_table.scss

```diff
@@ -36,3 +36,180 @@ table{
 		}
 	}
 }
+
+.sakai-table-toolBar
+{
+	@include display-flex();
+	@include align-items(flex-end);
+	margin: 0;
+
+	&+table
+	{
+		margin-top: 0;
+		padding-top: 0;
+	}
+
+	.sakai-table-filterContainer
+	{
+		@include display-flex();
+		@include flex-direction(column);
+		margin: 0 auto 10px 0;
+		padding-right: 10px;
+
+		label
+		{
+			margin: 0 4px 0 0;
+		}
+	}
+	.sakai-table-viewFilter, .sakai-table-searchFilter
+	{
+		@include display-flex();
+		@include align-items(center);
+	}
+	.sakai-table-viewFilter ~ .sakai-table-searchFilter
+	{
+		margin-top: 10px;
+	}
+	.sakai-table-searchFilter
+	{
+		@include flex-grow(1);
+
+		.sakai-table-searchFilter-searchField
+		{
+			margin-right: 4px;
+			width: 200px;
+			@include flex-grow(1);
+		}
+
+		input[type=button],	input[type=submit],
+		input[type=button]:disabled, input[type=submit]:disabled,
+		button,	button:disabled
+		{
+			margin: 0 4px 0 0;
+		}
+	}
+
+	.sakai-table-pagerContainer
+	{
+		margin-bottom: 10px;
+		text-align: center;
+		margin-left: auto;
+		padding-left: 10px;
+	}
+	.sakai-table-pagerLabel
+	{
+		padding: 0 5px 3px 5px;
+		font-size: 12px;
+		color: #505050;
+		text-align: center;
+	}
+	.sakai-table-pagerControls
+	{
+		@include justify-content(center);
+		padding: 0;
+
+		label
+		{
+			display: none;
+		}
+
+		input, button
+		{
+			font-size: 12px;
+			line-height: 1;
+
+			&:first-of-type
+			{
+				margin-left: 0;
+			}
+		}
+
+		input, select, button
+		{
+			margin: 0 0 0 10px;
+		}
+
+	}
+
+	.sakai-table-toolBar-centre
+	{
+		margin-bottom: 10px;
+	}
+
+	.sakai-table-viewFilter select
+	{
+		width: 100%;
+	}
+
+	.sakai-table-searchFilterControls
+	{
+		@include display-flex();
+		width: 100%;
+
+		input:last-of-type
+		{
+			margin-right: 0;
+		}
+	}
+
+	@media #{$tablet}
+	{
+		@include flex-direction(column);
+		@include justify-content(flex-start);
+		@include align-items(center);
+
+		.sakai-table-filterContainer, .sakai-table-viewFilter
+		{
+			margin-right: 0;
+			padding-right: 0;
+		}
+
+		.sakai-table-pagerContainer
+		{
+			margin-left: 0;
+			padding-left: 0;
+		}
+
+		.sakai-table-searchFilter .sakai-table-searchFilter-searchField
+		{
+			width: 150px;
+		}
+	}
+
+	@media all and (max-width: 400px)
```

Comment:
I would add another media variable on `_defaults.scss` just in case anyone else needs it.

Reply:
`smallPhone` or something like that

Reply:
This breakpoint is here specifically to allow the pager buttons to stack when there isn't enough room. 400px was chosen because of the size of the pager controls, so I don't know if it makes sense to base something general like "smallPhone" on this value.